run_kraken.py

```diff
@@ -362,12 +373,16 @@ def main(cfg) -> int:
                                 break
 
             iteration += 1
+            health_checker.current_iterations += 1
 
         # telemetry
         # in order to print decoded telemetry data even if telemetry collection
         # is disabled, it's necessary to serialize the ChaosRunTelemetry object
         # to json, and recreate a new object from it.
         end_time = int(time.time())
+        health_check_worker.join()
+        health_check_telemetry = health_check_telemetry_queue.get()
```

Comment:
Tests are failing because the queue is empty and the `get()` method will wait forever. Please use `get_nowait()` instead catching `Empty` exception.